README.md

```diff
@@ -70,6 +70,8 @@ The language itself draws pieces of influence from:
 
 # Contributors
 
+Information and instructions for potential contributors are provided [here](CONTRIBUTING).
+
```

Comment:
The information on the pre-commit hook should be moved to the existing `.github/CONTRIBUTING.md` file as well.

Reply:
Would still like this

Reply:
Robin, the CONTRIBUTING.md page refers to a wiki page about the process for contributing PRs, which also refers to https://github.com/dafny-lang/dafny/wiki/INSTALL#building-and-developing-from-source-code,which is the one location for all dependencies, including the pre-commit installation (which is only needed if one is going to propose a PR). So do you really want to call out one dependency, the pre-commit script, for special mention?

Reply:
Ah I didn't realize it was also documented there. In that case let's just remove this copy of the content about the pre-commit entirely.